app/scraper.py

```diff
@@ -95,6 +96,8 @@ async def scrape(
     tracer_id: str,
     scraped_data_repository: ScrapedDataRepository,
     telegram_client: TelegramClient,
+    start_date=start_date,
```

Comment:
Here I get an error because: (a) this syntax (`... = ...`) in a function signature is assigning default values, but then after these come other arguments (`openai_api_key` and `log_level`) without default values, and this raises a syntax error (at least in python 3.11); (b) these default values are not defined at this point.

What I did to make the scraper run with your changes was to change these two lines to:
```python
start_date: str,
end_date: str,
```
but you could fix this differently (e.g., by dynamically generating a default end_date that is the date of "today" or something like that)